specs/binding-tls.spec/src/main/resources/META-INF/zilla/tls.idl

```diff
@@ -23,7 +23,20 @@ scope tls
             TLS_PROTOCOL_REJECTED (2),
             TLS_KEY_REJECTED (3),
             TLS_PEER_NOT_VERIFIED (4),
-            TLS_HANDSHAKE_FAILED (5)
+            TLS_HANDSHAKE_FAILED (5),
+            TLS_KEY_PAIR_VERIFICATION_FAILED (6)
```

Comment:
Even though the implementation detail is a key pair, the syntax in `zilla.yaml` is just `keys`, so let's name the event using key instead of key pair too.
```suggestion
            TLS_KEY_VERIFICATION_FAILED (6)
```

Note: this feedback applies to all naming in this PR.